app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -31,42 +31,50 @@ object FeatureFlags {
     /**
      * Enables the Home button in the browser toolbar to navigate back to the home screen.
      */
-    val showHomeButtonFeature = Config.channel.isNightlyOrDebug || Config.channel.isBeta
+    @Suppress("MayBeConst")
```

Comment:
You can make these `const val` too. See the feature line above.

Reply:
🙌🏽  Updated :)